gnosis/verification.k

```diff
@@ -6,21 +6,35 @@ module VERIFICATION
     imports EDSL
     imports LEMMAS
 
+  // ########################
+  // Memory Usage
+  // ########################
+
+    rule #memoryUsageUpdate(#memoryUsageUpdate(MU, START0, WIDTH0), START1, WIDTH1) => #memoryUsageUpdate(MU, START0, WIDTH0)
+      requires START1 +Int WIDTH1 <Int START0 +Int WIDTH0
+
+    rule #memoryUsageUpdate(#memoryUsageUpdate(MU, START0, WIDTH0), START1, WIDTH1) => #memoryUsageUpdate(MU, START1, WIDTH1)
+      requires START0 +Int WIDTH0 <=Int START1 +Int WIDTH1
```

Comment:
for both rules, i think you should reason about the case when WIDTH0 or WIDTH1 is zero.

my suggestion (not sure if works though):
- adding WIDTH0 >Int 0 andBool WIDTH1 >Int 0
- adding another rule:
```
rule #memoryUsageUpdate(MU, START, WIDTH) => MU  when WIDTH ==Int 0
```
or change the semantic rule directly:
https://github.com/kframework/evm-semantics/pull/281/files#diff-1e76ae9645a9718bd96a177a24eb3865R2000

Reply:
done. see https://github.com/kframework/evm-semantics/commit/37a90febd2e727e4fded639e6cada074050cc5e4